iota-client/src/client.rs

```diff
@@ -232,7 +232,7 @@ impl std::fmt::Debug for Client {
         d.field("sync", &self.sync).field("client", &self.client);
         #[cfg(feature = "mqtt")]
         d.field("broker_options", &self.broker_options);
-        d.field("local_pow", &self.local_pow).finish()
+        d.field("local_pow", &self.network_info.local_pow).finish()
```

Comment:
should we dump the whole network_info object here instead of just the local_pow field?

Reply:
Makes sense